x/auth/client/cli/broadcast.go

```diff
@@ -24,20 +24,27 @@ filename, the command reads from standard input.
 $ <appcli> tx broadcast ./mytxn.json
 `),
 		Args: cobra.ExactArgs(1),
-		RunE: func(cmd *cobra.Command, args []string) (err error) {
+		RunE: func(cmd *cobra.Command, args []string) error {
 			cliCtx := context.NewCLIContext().WithCodec(cdc)
 			stdTx, err := client.ReadStdTxFromFile(cliCtx.Codec, args[0])
 			if err != nil {
-				return
+				return err
 			}
 
 			txBytes, err := cliCtx.Codec.MarshalBinaryBare(stdTx)
 			if err != nil {
-				return
+				return err
 			}
 
 			res, err := cliCtx.BroadcastTx(txBytes)
-			cliCtx.PrintOutput(res)
+			if err != nil {
+				return err
+			}
+
+			err = cliCtx.PrintOutput(res)
```

Comment:
```suggestion
			return cliCtx.PrintOutput(res)
```

You can then remove all the rest